lib/event_tracer/dynamo_db_log_worker.rb

```diff
@@ -0,0 +1,60 @@
+# frozen_string_literal: true
+
+require 'sidekiq'
+module EventTracer
+  class DynamoDBLogWorker
+    include ::Sidekiq::Worker
+
+    sidekiq_options retry: 1, queue: 'low'
+
+    TABLE_NAME = ENV.fetch('AWS_DYNAMODB_LOGGING_TABLE', 'logs')
+    # See https://docs.aws.amazon.com/sdk-for-ruby/v3/api/Aws/DynamoDB/Client.html#batch_write_item-instance_method
+    MAX_DYNAMO_DB_ITEM_PER_REQUEST = 25
```

Comment:
do we want this configurable too?

Reply:
We probably shouldn't? Or we set it at a max of 25? This one isn't so helpful for the apps, unless they somehow have a huge logging payload.